add import and export in index (#39)
fewlinesco/connect-js: client/CHANGELOG.md

All notable changes to this project will be documented in this file.

> 
> ## [0.4.5] - 2021-05-05
> 

Reviewer: ```suggestion

## [0.4.6] - 2021-05-05

```

> 
> - Added the export of `generateJWE`
> 

Reviewer: ```suggestion

- Added the export of `generateJWE`.

```

> - Added the export of `generateJWE`. This was missing from `0.4.4`

Reviewer 2: ```suggestion
- Added the export of `generateJWE`. This was missing in the `0.4.4` version.
```
You were too quick to make the change ;)

Author: 😄

All notable changes to this project will be documented in this file.

## [0.4.6] - 2021-05-05

> - Added the export of `generateJWE`.

Reviewer 2: ```suggestion
- Added the export of `generateJWE`. This was missing from `0.4.4`
```
Otherwise it could raise questions. This way we give all the need information about why this change is needed.
